src/main/java/bookstore/controller/BookController.java

```diff
@@ -48,7 +49,7 @@ public void deleteById(@PathVariable Long id) {
 
     @ResponseStatus(HttpStatus.OK)
     @PutMapping("/{id}")
-    public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestDto
+    public BookDto updateBook(@PathVariable Long id, @RequestBody @Valid CreateBookRequestDto
```

Comment:
just FYI

you may also validate such things as @PathVariable, @RequestParams etc. It looks like:

`@PathVariable @Positive Long id`
In order to be able to perform such validation you need to annotate your controller class with Validated annotation

again, this is just FYI - apply it or not :)